ppml/trusted-big-data-ml/python/docker-gramine/java-manifest/init.sh

```diff
@@ -26,12 +26,9 @@ else
     echo "both /dev/sgx/provision /dev/sgx_provision are not ready, please check the kernel and driver"
 fi
 
-if [ -c "/ppml/trusted-big-data-ml/secured_argvs" ]; then
-    echo "/ppml/trusted-big-data-ml/secured_argvs is ready"
-else
-    echo "/ppml/trusted-big-data-ml/secured_argvs is not ready, please generate it before init.sh"
-fi
-
 ls -al /dev/sgx
 
-make SGX=1 DEBUG=1 THIS_DIR=/ppml/trusted-big-data-ml  SPARK_LOCAL_IP=$local_ip SPARK_USER=root G_SGX_SIZE=$sgx_mem_size G_LOG_LEVEL=$sgx_log_level
+/graphene/Pal/src/host/Linux-SGX/signer/pal-sgx-get-token -output /ppml/trusted-big-data-ml/bash.token -sig /ppml/trusted-big-data-ml/bash.sig
```

Comment:
This does not seem to be the operation of the Gramine

Reply:
Fixed